napari/components/dims.py

```diff
@@ -193,7 +193,7 @@ def set_range(
             Sequence[Union[int, float]], Sequence[Sequence[Union[int, float]]]
         ],
         *,
-        restore_point: Optional[Sequence[float]] = None,
+        restore_point: Optional[Tuple[float, ...]] = None,
```

Comment:
While this function's implementation should work with any `Sequence`, we only need it to work a `Tuple` for now.